.github/CODEOWNERS

```diff
@@ -75,10 +75,8 @@
 # ==== Libraries and frameworks ====
 
 # Ray data.
-/python/ray/data/ @ericl @scv119 @clarkzinzow @jjyao @jianoaix @c21
-/python/ray/data/preprocessors/ @clarkzinzow @jiaodong @Yard1 @bveeramani @matthewdeng @amogkam
```

Comment:
I don't think we need a special group for `preprocessors` sub-directory, so I removed them here.